components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js

```diff
@@ -0,0 +1,327 @@
+                        });
+                        thisTable.addClass("table-selectable");
+                        $(button).addClass("active").html('Cancel');
+                        $(button).closest('li').siblings('.select-all-btn').show();
+                    } else if ($(button).html() == 'Cancel'){
+                        thisTable.api().rows().every(function () {
+                            $(this.node()).removeAttr('data-type');
+                            $(this.node()).removeClass(ROW_SELECTED_CLASS);
+                        });
+                        thisTable.removeClass("table-selectable");
+                        $(button).removeClass("active").html('Select');
+                        $(button).closest('li').siblings('.select-all-btn').hide();
+                    }
+                });
+
+                //Select/Deselect all rows functions
+                $('.dataTables_wrapper').off('click', '[data-click-event=toggle-select-all]');
+                $('.dataTables_wrapper').on('click', '[data-click-event=toggle-select-all]', function () {
+                    var button = this,
+                        thisTable = $(this).closest('.dataTables_wrapper').find('.dataTable').dataTable();
+                    if ($(button).html() == 'Select All') {
+                        thisTable.api().rows().every(function () {
+                            $(this.node()).addClass(ROW_SELECTED_CLASS);
+                            $(button).html('Deselect All');
+                        });
+                    } else if ($(button).html() == 'Deselect All') {
+                        thisTable.api().rows().every(function () {
+                            $(this.node()).removeClass(ROW_SELECTED_CLASS);
+                            $(button).html('Select All');
+                        });
+                    }
+                });
+
+
+                //Event for row select/deselect
+                $('body').on('click', '[data-type=selectable]', function() {
+                    $(this).toggleClass(ROW_SELECTED_CLASS);
+                    var button = this,
+                        thisTable = $(this).closest('.dataTables_wrapper').find('.dataTable').dataTable();
+
+                    thisTable.api().rows().every(function () {
+                        if (!$(this.node()).hasClass(ROW_SELECTED_CLASS)) {
+                            $(button).closest('.dataTables_wrapper').find('[data-click-event=toggle-selected]').html('Select All');
+                        }
+                    });
+                });
+
+                //list table list/grid view toggle function
+                var toggleButton = $('[data-click-event=toggle-list-view]');
+                toggleButton.click(function () {
+                    if ($(this).attr('data-view') == 'grid') {
+                        $(this).closest('.dataTables_wrapper').find('.dataTable').addClass('grid-view');
+                        //$(this).closest('li').hide();
```

Comment:
@erangatl shall we remove the commented lines ?

Reply:
Fixed with #1690